docs/bugs.json

```diff
@@ -1,14 +1,22 @@
 [
     {
+        "name": "DelegateCallReturnValue",
+        "summary": "The low-level .delegatecall() takes the returned value as a boolean instead of the execution outcome.",
```

Comment:
What about `... does not return the execution outcome but converts the value returned by the function call to a boolean instead.`